taskcluster/ac_taskgraph/transforms/beetmover.py

```diff
@@ -10,6 +10,7 @@
 from taskgraph.util.schema import resolve_keyed_by
 
 from ..build_config import get_version
+from .build import _get_nightly_version
```

Comment:
Nit: Let's make `_get_nightly_version()` public, if we get to import it from another module.